src/popup/router/pages/Networks.vue

```diff
@@ -59,6 +59,16 @@
         v-model="network.compilerUrl"
         data-cy="compiler"
       />
+      <div class="text-left expand" @click="expanded = !expanded">
+        <img :class="expanded ? 'open' : ''" src="../../../icons/carret-down.svg" />
+        <span>{{ expanded ? $t('pages.network.showLess') : $t('pages.network.showMore') }}</span>
```

Comment:
I suggest we have something in the way of `Tipping configuration` or something like that instead of the `Show more/Show less` as this does not bring much information to the user e.g. what is this field for.